### PR TITLE
Restore overhaul

### PR DESCRIPTION
**Pull request by Tejeev**
I've made a LOT of changes here and I think this still needs a bit of an overhaul.  We may want to review this some time soon.

### CODE REVIEW

File: content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ aliases:
 This procedure describes how to use RKE to restore a snapshot of the Rancher Kubernetes cluster. 
 This will restore the Kubernetes configuration and the Rancher database and state.
 
+> **NOTE** This doc says to change the name of the cluster yaml and rke state file.  If you have version control in place or some other method of backing up your important files, this may not be necessary.  Adapt as needed.
```
Comment (superseb) [inline content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md:12]: This is unclear, we should instruct the known good way. If there is an exception that needs to be described, describe it in detail (when does it apply and what does it change). Adapt as needed does not help the reader.

File: content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md [hunk 2]
```diff
@@ -19,7 +21,8 @@ RKE v0.2.0 has changed how the [Kubernetes cluster state is stored]({{<baseurl>}
 - [2. Place Snapshot](#2-place-snapshot)
 - [3. Configure RKE](#3-configure-rke)
 - [4. Restore Database](#4-restore-database)
-- [5. Bring Up the Cluster](#5-bring-up-the-cluster)
+<!-- - [5. Bring Up the Cluster](#5-bring-up-the-cluster) -->  <!-- This section does not appear to be necessary as the snapshot restore will run rke up -->
```
Comment (superseb) [inline content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md:24]: Just remove unneeded sections, we have version control to see changes. I guess this is related to the pre v0.2 way of doing things? In some parts of this doc, we do describe 0.2+ changes, is this doc supposed to cover pre 0.2 and post 0.2 situations?

File: content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md (hunk 1 above)
Comment (janeczku) [reply content/rancher/v2.x/en/backups/restorations/ha-restoration/_index.md:12]: We should instruct user to backup the config and rkestate but modify and use the **existing** files.

**Comment by catherineluse** [comment]
relevant issue https://github.com/rancher/docs/issues/2590

**Comment by janeczku** [comment]
@Tejeev I suggest we address the issues in the linked issue in this PR. Let me know if you want to have a quick chat about that.

**Comment by Tejeev** [comment]
Solved by Merge

**Pull request closed without merging**